addons/dialogue_manager/components/download_update_panel.tscn

```diff
@@ -46,14 +46,14 @@ layout_mode = 2
 [node name="DownloadButton" type="Button" parent="VBox/Center"]
 unique_name_in_owner = true
 layout_mode = 2
-text = "Download update"
+text = "Descargar actualización"
```

Comment:
I guess you didn't intend to change all these files in `addons/dialogue_manager`. You should revert them:

```console
$ git restore --source=6c579a4a257abdc5f23b5ac09a0a98da88c191b5 addons/dialogue_manager
$ git add addons/dialogue_manager
$ git commit -m 'Revert unintended changes to Dialogue Manager addon'
```